o2a/o2a_libs/el_wf_functions.py

```diff
@@ -14,24 +14,68 @@
 # limitations under the License.
 """All WF EL functions"""
 
+from typing import Optional
 
-def wf_app_path():
+from jinja2 import contextfunction
+
+from airflow.models import TaskInstance, DagRun
+from airflow.utils.db import provide_session
+from airflow import AirflowException
+
+
+def conf(key: str) -> str:
     """
-    It returns the workflow application path for the current workflow job.
+    It returns the value of the workflow job configuration property for the
+    current workflow job, or an empty string if undefined.
+
     This has the effect that some parameters cannot be templated, and thus
     this will fail.
     """
+    return "params[{}]".format(key)
 
 
-def wf_conf(key: str) -> str:
+@contextfunction
+@provide_session
+def last_error_node(context=None, session=None) -> str:
```

Comment:
Those two decorators provide access to Airflow SQLAlchemy session and context variables. According to tests it works but will be verified when mappers will use new translations.

Reply:
Nice!